homeassistant/components/unmanic/sensor.py

```diff
@@ -0,0 +1,126 @@
+"""Sensor platform for Unmanic."""
+from homeassistant.components.sensor import SensorEntity, SensorEntityDescription
+from homeassistant.helpers.typing import StateType
+
+from .const import DOMAIN, NAME
+from .entity import UnmanicEntity
+
+SENSORS: tuple[SensorEntityDescription, ...] = (
+    SensorEntityDescription(
+        key="total-workers",
+        name=f"{NAME} Total Workers",
```

Comment:
If you want to add `NAME` to all entities, I think it is better to just do this in your entity itself.

```self._attr_name = f"{NAME} {self.entity_description.name}"```